src/main/scala/it/pagopa/interop/apigateway/common/ApplicationConfiguration.scala

```diff
@@ -15,7 +15,9 @@ object ApplicationConfiguration {
   val attributeRegistryManagementURL: String =
     config.getString("interop-api-gateway.services.attribute-registry-management")
   val purposeManagementURL: String           = config.getString("interop-api-gateway.services.purpose-management")
-  val notifierURL: String                    = config.getString("interop-api-gateway.services.notifier")
+  val tenantProcessURL: String               = config.getString("interop-api-gateway.services.tenant-process")
```

Comment:
Servirà sta roba nel deployment

Reply:
anzi, non mi sembra di vederle nemmeno nell'application.conf di test e main :(

Reply:
si, bisogna aggiungerla nel deployment.
nell'application.conf di test non serve, in quello del main l'ho aggiunto [qui](https://github.com/pagopa/interop-be-api-gateway/pull/94)

Reply:
sì, ho fatto la review di quella solo dopo.